subiquity/ui/views/mirror.py

```diff
@@ -227,17 +228,19 @@ async def confirm_continue_anyway() -> None:
             )
 
             if confirmed:
-                self.controller.done(result.url.value)
+                skip_archive = True if continue_skip_archive else None
+                self.controller.done(result.url.value, skip_archive=skip_archive)
 
         log.debug("User input: {}".format(result.as_data()))
         if self.has_network and self.last_status in [
             MirrorCheckStatus.RUNNING,
             MirrorCheckStatus.FAILED,
             None,
         ]:
-            async_helpers.run_bg_task(confirm_continue_anyway())
+            status_is_failed = bool(self.last_status == MirrorCheckStatus.FAILED)
```

Comment:
nitpick: `bool()` is redundant

Reply:
Well... black does not like the parentheses so I guess that's why I used bool() in the first place. Will do `status_is_failed = self.last_status == MirrorCheckStatus.FAILED` then